roles/privacyidea_ha/hacluster.yml

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
This file can be deleted, it's a duplicate of roles/privacyidea_ha/tasks/hacluster.yml

Reply:
File removed